Assets/Scripts/PackItUp/MockSystems/MockLevel.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using PackItUp.Interactables;
+using PackItUp.Managers;
+using UnityEngine;
+
+namespace PackItUp.MockSystems {
+    
+    /// <summary>
+    /// This mock level implementation will fetch all EndZones added to the scene.
+    /// </summary>
+    public class MockLevel : MonoBehaviour, ILevel
+    {
+        [SerializeField] private EndZone[] _endZones;
+        private List<object> _items;
+
+        private void Awake() {
+            _endZones = FindObjectsByType<EndZone>(FindObjectsInactive.Exclude, FindObjectsSortMode.None);
```

Comment:
This can be left up for design choices but I don't think we will have multiple end zones on a level